src/styles/navbar.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__logo {
+    display: block;
+  }
+
+  &__nav {
+    padding: 10px 0;
+
+    &-items {
+      padding: 0;
+      display: flex;
+      align-items: center;
+      text-transform: uppercase;
+      font-size: 12px;
+      font-weight: 500;
+    }
+
+    &-item {
+      text-align: center;
+      list-style: none;
+      margin: 0 10px;
+
+      &:first-child {
+        margin-left: 0;
+      }
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &-link {
+      text-decoration: none;
+      padding: 22.5px 0;
+      color: $primary-color-text;
+      transition: color 0.5s;
+
+      &:hover {
+        color: $main-color;
+      }
+    }
+  }
+}
+
+[data-qa='nav-hover']:hover {
+  color: $main-color;
+}
+
+a.is-active {
+  position: relative;
+  color: $main-color;
```

Comment:
The variable `$main-color` is used here but not defined in this file. Ensure it is defined or imported from another SCSS file to avoid compilation errors.